### PR TITLE
Automagically generate cassette names

### PR DESCRIPTION
**Pull request by XanderStrike**
Use the `use_vcr` block to automatically generate a cassette name for a test based on the file and test name calling the block.

### CODE REVIEW

**Comment by bboe** [comment]
Love it!

File: test/helper.rb [hunk 1]
```diff
@@ -12,6 +12,18 @@
...
   config.filter_sensitive_data('http://localhost') { ENV['RENTLINX_SITE_URL'] }
+
+  config.before_record do |i|
+    i.request.headers.delete('Authentication-Token')
+  end
+end
+
+def use_vcr
+  cassette_name =  caller[0][%r{/.*\.}].split('/').last[0..-2] +
+                   '-' + caller[0][/`.*'/][1..-2]
```
Comment (scuendet) [inline test/helper.rb:25]: this could use a comment, I think

**Comment by scuendet** [comment]
:+1:

**Comment by d-ray** [comment]
:+1:

**Comment by XanderStrike** [comment]
:+1:

**Pull request merged**